contrib/beginner_example/inputs.json

```diff
@@ -1,3 +1,3 @@
 {
-  "bam_chrom_counter.bam" : "dx://project-BfZj3KQ03zx77KF7pjjBFZkf:/testexomes/testexome.pair0.tumour.bam"
+  "bam_chrom_counter.bam" : "dx://project-BzQf6k80V3bJk7x0yv6z82j7:file-Fvy3pY00Zx46zQF39Bj9FVyF"
```

Comment:
It is a bam file in "DNAnexus Regression Testing Project AWS US east" on PROD. Shall we provide a sample input file here? If so is there a project open to all users that we can dump a sample bam? 
(There is no .bam in "Reference Genome Files: AWS US (East)".)

Reply:
Both the previous and this file will not be available to general public. I think we should find a public project and place this file there. I'll share such a project with you.

Reply:
https://platform.dnanexus.com/panx/projects/BQbJpBj0bvygyQxgQ1800Jkk/data/Developer%20Quickstart is public